lib/elixir/test/elixir/io/ansi_test.exs

```diff
@@ -86,6 +86,13 @@ defmodule IO.ANSITest do
            "#{IO.ANSI.red}Hello!"
   end
 
+  test "format with color number" do
+    data = [IO.ANSI.color(111), "Hello", IO.ANSI.color_background(222), "World"]
```

Comment:
I think we could skip `IO.chardata_to_string` and `IO.ANSI.format` (they are covered via other tests) and do `IO.ANSI.color` testing directly.